### DISCUSSION TITLE
Test scenario that opens request details in new tab

### DISCUSSION THREAD

**Discussion by engyedward** [cypress-io/cypress#17845] [Questions and Help]
I'm working on an application that has a module named "My Workspace", where the end user is able to view a big list of "Customer Requests" with wide range of filtration options above the list. 
When I click a row (request) in this grid, the request details is opened in a new tab. it is done like this to avoid losing the search criteria that this user might have done.
The tr doesn't hold any href for the request details page. All is in the angular code.
is there any way to handle this case with cypress? or i have to use another tool?

![image](https://user-images.githubusercontent.com/89347481/130358780-5181b8e0-944e-487c-80d3-e01deea100ef.png)

**Comment by herecydev** [comment]
This sounds like a bad idea, how would a user, especially using a screen reader, know this is a link? If it had a href it would be testable and solve what I mentioned.
